tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs

```diff
@@ -455,6 +457,12 @@ private static bool VerifyAssignCellToAVCSUnitOfSale(string childCell, string pr
                             AttrNotMatched.Add(nameof(item.PRODUCT));
                         if (!item.PRODTYPE.Equals((GetProductInfo(unitOfSale.CompositionChanges.AddProducts)).ProductType))
                             AttrNotMatched.Add(nameof(item.PRODTYPE));
+                        if (!item.WEEKNO.Equals(weekNoTag))
```

Comment:
Please re-order the assertions or move the comment.

Reply:
Move the comment section up.